core/products/future_for_test.go

```diff
@@ -16,11 +16,14 @@ import (
 	"context"
 	"strconv"
 
+	datapb "code.vegaprotocol.io/vega/protos/vega/data/v1"
+
 	"code.vegaprotocol.io/vega/core/oracles"
 )
 
-func (f *Future) SetSettlementData(ctx context.Context, priceName string, settlementData uint64) {
+func (f *Future) SetSettlementData(ctx context.Context, priceName string, settlementData uint64, settlementDataType datapb.PropertyKey_Type) {
```

Comment:
Should this function still take a `uint64`? If settlement data is a decimal, then this doesn't really make much sense

Reply:
Yes, I missed this, has been changed now
Thanks!